### DISCUSSION TITLE
Support Render Streaming

### DISCUSSION THREAD

**Discussion by FullstackJack** [flareact/flareact#237] [Ideas]
I noticed that we are using `renderToString()`. Any reason why `renderToReadableStream()` hasn't be used here? Am I missing a stream being used some place else? Is there a limitation to this implementation or has it just not been thought of yet? It looks like Cloudflare Workers would support this feature since `Response` can take a stream. Seems like an easy win for time-to-first-byte (TTFB).

https://developers.cloudflare.com/workers/learning/using-streams/
https://reactjs.org/docs/react-dom-server.html#rendertoreadablestream

**Comment by traffisco** [comment]
renderToReadableStream needs to be ported to cloudflare stream type.

#122
